gate-basic/src/main/groovy/com/netflix/spinnaker/gate/security/basic/BasicAuthConfig.groovy

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Please refrain from introducing new Groovy files in the `main` source set, as the spinnakerbot called out.